apikana-maven-plugin/src/main/java/org/swisspush/apikana/AbstractApikanaMojo.java

```diff
@@ -26,11 +27,12 @@
 import java.util.function.Consumer;
 import java.util.jar.JarEntry;
 import java.util.jar.JarFile;
+import org.yaml.snakeyaml.Yaml;
 
 import static org.twdata.maven.mojoexecutor.MojoExecutor.*;
 
 public abstract class AbstractApikanaMojo extends AbstractMojo {
-    protected final static String OUTPUT = "target/api";
+    protected final static String OUTPUT = "target/node/dist";
```

Comment:
The implication is that we will have a structure in target/node that is very close to what would be generated by `apikana init`. It does not change anything in regards to the JARs. For the JARs it does not matter, where the artifacts are inside `target` since `OUTPUT` is always used to add them. See eg. [GenerateMojo](https://github.com/swisspost/apikana-java/blob/master/apikana-maven-plugin/src/main/java/org/swisspush/apikana/GenerateMojo.java#L154)